hackathon_site/hardware/test_api.py

```diff
@@ -1708,3 +1743,108 @@ def test_failed_beginning_status(self):
             response.json(), {"status": ["Cannot change the status for this order."]},
         )
         self.assertFalse(request_data["status"] == Order.objects.get(id=self.pk).status)
+
+
+class MinMaxTeamOrderTestCase(SetupUserMixin, APITestCase):
```

Comment:
Nice job for creating a new class! But we should put tests that belong to the same view in the same class. In your case you changed the OrderCreateSerializer which belongs to the OrderListViewPostTestCase class, could you move all your tests there. 

If it helps you can create more helper functions and create users/profiles in a for loop if the code gets too bloated.